app/models/ability.rb

```diff
@@ -0,0 +1,40 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Define abilities for the user here. For example:
+    #
+    return unless user.present?
+
+    can :read, :all
```

Comment:
- [ ] setting your app up like this the user that created the post or comment isn't able to delete it.

Reply:
Resolved

Reply:
well resolved! ✔️